cmd/oras/internal/display/metadata/tree/discover.go

```diff
@@ -20,6 +20,7 @@ import (
 	"io"
 	"strings"
 
+	"github.com/fatih/color"
```

Comment:
ORAS already leverages [github.com/morikuni/aec](https://pkg.go.dev/github.com/morikuni/aec@v1.0.0) for coloring so we don't need to introduce a new dependency. We can refer to the existing implementation in `status.go`:
https://github.com/oras-project/oras/blob/dfd8b1e42e799617d9222b7833fc53914b23737c/cmd/oras/internal/display/status/progress/status.go#L36-L41